.gitignore

```diff
@@ -91,3 +91,4 @@ utils/starmap/*.svg
 utils/starmap/*.png
 utils/starmap/*.jpg
 utils/starmap/*.log
+/compile_commands.json
```

Comment:
shouldn't this be a relative path?

Reply:
Maybe? That's the only spot it should be showing up though